pulpcore/cli/common/context.py

```diff
@@ -557,6 +557,16 @@ def modify(
         )
 
 
+class PulpContentContext(PulpEntityContext):
+    """
+    Base class for content specific contexts
+    """
+
+    ENTITY = "content"
+    ENTITIES = "content"
+    LIST_ID = "content_list"
```

Comment:
```suggestion
    LIST_ID = "content_list"
    UNIQUE_LOOKUPS: ClassVar[List[str]] = []
```
I still think we should call it `NATURAL_KEY` or `NATURAL_KEY_NAMES`.